python/GafferUI/GraphEditor.py

```diff
@@ -298,7 +299,8 @@ def __keyPress( self, widget, event ) :
 		elif event.key == "Down" :
 			selection = self.scriptNode().selection()
 			if selection.size() == 1 and selection[0].parent() == self.graphGadget().getRoot() :
-				needsModifiers = not Gaffer.Metadata.value( selection[0], "graphEditor:childrenViewable" )
+				## \todo: Remove nodeGraph fallback when all client code has been updated
+				needsModifiers = not Gaffer.Metadata.value( selection[0], "graphEditor:childrenViewable" ) or not Gaffer.Metadata.value( selection[0], "nodeGraph:childrenViewable" )
```

Comment:
This stops the down arrow working with Boxes. If metadata doesn't exist then `value()` returns None, so one of these is always going to be `None`. I suggest moving the query into a `__childrenViewable( node )` utility method and calling it both here and in `appendContentsMenuDefinitions()`. If "graphEditor:childrenViewable" is non-None, we should return it, then if "nodeGraph:childrenViewable" is non-None we should return that, and finally we should just return None.

Reply:
Addressed in df0f84813d5cc05d6dfb05960a4103bf56a7b6c9